README.md

```diff
@@ -11,7 +11,7 @@
 This exporter allows exporting metrics created using the [OpenTelemetry SDK for JavaScript](https://github.com/open-telemetry/opentelemetry-js)
```

Comment:
Should we get rid of the disclaimer above? 
> This exporter is based on the OpenTelemetry Metrics SDK for JavaScript,
> which is currently in an RC state and neither considered stable nor
> complete as of this writing.
> As such, this exporter is not intended for production use until the
> underlying OpenTelemetry Metrics API and SDK are stable.
> See [open-telemetry/opentelemetry-js](https://github.com/open-telemetry/opentelemetry-js)
> for the current state of the OpenTelemetry SDK for JavaScript.

Reply:
Removed the statement about OTel JS being in RC state, and added a statement that this exporter is in RC state and not ready for production use in [042f6e7](https://github.com/dynatrace-oss/opentelemetry-metric-js/pull/38/commits/042f6e7840dec547c4418eb0f99c603af59e07cb).

We can remove the statement altogether when we release 1.0.0 of this exporter :slightly_smiling_face: